nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenStat.scala

```diff
@@ -258,6 +258,7 @@ trait NirGenStat { self: NirGenPhase =>
             case ann if ann.symbol == NoInlineClass   => Attr.NoInline
             case ann if ann.symbol == InlineHintClass => Attr.InlineHint
             case ann if ann.symbol == InlineClass     => Attr.AlwaysInline
+            case ann if ann.symbol == StubClass       => Attr.Stub
```

Comment:
Lets also support `@stub` on classes and modules, this should be trivial by adding similar line to `genClassAttrs`.